src/styles/MuiThemeProvider.js

```diff
@@ -84,8 +85,6 @@ export const MUI_SHEET_ORDER = [
   'MuiCardHeader',
   'MuiCard',
 
-  'MuiTextFieldLabel',
-  'MuiTextFieldInput',
```

Comment:
There should probably be unit tests that ensures that there are no dead style sheets. Considering ssr is possible, it should be doable.